src/components/Button/Button.tsx

```diff
@@ -107,12 +110,30 @@ const Button = forwardRef<HTMLButtonElement, ButtonProps>(function Button(
     },
   }
 
+  const spinnerColors: Record<string, ThemeUICSSObject> = {
```

Comment:
C: Can the performance be improved by wrapping this variable assignment with the useMemo?

Reply:
Reworked it using useMemo